Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -4517,6 +4522,248 @@ script:
+    - contextPath: MicrosoftATP.Machine.ComputerDNSName
+      description: The machine DNS name.
+      type: String
+    - contextPath: MicrosoftATP.Machine.FirstSeen
+      description: The first date and time where the machine was observed by Microsoft Defender ATP.
+      type: Date
+    - contextPath: MicrosoftATP.Machine.LastSeen
+      description: The last date and time where the machine was observed by Microsoft Defender ATP.
+      type: Date
+    - contextPath: MicrosoftATP.Machine.OSPlatform
+      description: The operating system platform.
+      type: String
+    - contextPath: MicrosoftATP.Machine.OSVersion
+      description: The operating system version.
+      type: String
+    - contextPath: MicrosoftATP.Machine.OSProcessor
+      description: The operating system processor.
+      type: String
+    - contextPath: MicrosoftATP.Machine.LastIPAddress
+      description: The last IP on the machine.
+      type: String
+    - contextPath: MicrosoftATP.Machine.LastExternalIPAddress
+      description: The last machine IP to access the internet.
+      type: String
+    - contextPath: MicrosoftATP.Machine.OSBuild
+      description: The operating system build number.
+      type: Number
+    - contextPath: MicrosoftATP.Machine.HealthStatus
+      description: The machine health status.
+      type: String
+    - contextPath: MicrosoftATP.Machine.RBACGroupID
+      description: The machine RBAC group ID.
+      type: Number
+    - contextPath: MicrosoftATP.Machine.RBACGroupName
+      description: The machine RBAC group name.
+      type: String
+    - contextPath: MicrosoftATP.Machine.RiskScore
+      description: The machine risk score.
+      type: String
+    - contextPath: MicrosoftATP.Machine.ExposureLevel
+      description: The machine exposure score.
+      type: String
+    - contextPath: MicrosoftATP.Machine.IsAADJoined
+      description: True if machine is AAD joined, False otherwise.
+      type: Boolean
+    - contextPath: MicrosoftATP.Machine.AADDeviceID
+      description: The AAD Device ID.
+      type: String
+    - contextPath: MicrosoftATP.Machine.MachineTags
+      description: Set of machine tags.
+      type: String
+    - contextPath: MicrosoftATP.Machine.IPAddresses.ipAddress
+      description: The machine IP address.
+      type: String
+    - contextPath: MicrosoftATP.Machine.IPAddresses.MACAddress
+      description: The machine MAC address.
+      type: String
+    - contextPath: MicrosoftATP.Machine.IPAddresses.operationalStatus
+      description: The machine operational status.
+      type: String
+    - contextPath: MicrosoftATP.Machine.IPAddresses.type
+      description: The machine macine IP address type.
+      type: String
+    - contextPath: MicrosoftATP.Machine.AgentVersion
+      description: The machine Agent version.
+      type: String
+  - arguments:
+    - default: false
+      description: 'A JSON object with list of MS defender ATP indicators to update. indicator_batch
```

Comment:
https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/ti-indicator?view=o365-worldwide add the link ref please.